### ISSUE TITLE
How do you deploy the Java proxy in tomcat?

### ISSUE THREAD

**Issue by ves-krd** [Esri/resource-proxy#508]

**Comment by ves-krd** [comment]
I have an apache-tomcat server running locally.  Do I need to convert the folder into a war and put it in the webapps folder?  Are there steps somewhere to follow?

**Comment by bsvensson** [comment]
@ves-krd No, you don't need to convert to war, see https://github.com/Esri/resource-proxy/tree/master/Java#instructions for installation instructions.
